metrics/config.go

```diff
@@ -144,13 +144,19 @@ func NewStackdriverClientConfigFromMap(config map[string]string) *StackdriverCli
 	}
 }
 
-// Record applies the `ros` Options to `ms` and then records the resulting
+// RecordN applies the `ros` Options to each measurement in `mss` and then records the resulting
 // measurements in the metricsConfig's designated backend.
-func (mc *metricsConfig) Record(ctx context.Context, ms stats.Measurement, ros ...stats.Options) error {
+func (mc *metricsConfig) RecordN(ctx context.Context, mss []stats.Measurement, ros ...stats.Options) error {
```

Comment:
Could you add a test for this function please?

Reply:
The function _is_ actually tested via Record. But yeah, as said in description — if we're pursuing this — I'll add tests :)